write-fonts/src/graph/splitting.rs

```diff
@@ -1066,4 +1063,81 @@ mod tests {
             .sum();
         assert_eq!(total_c2recs, expected_n_c2_recs);
     }
+
+    #[test]
+    fn size_of_value_record_children_sanity() {
+        // let's have  single class1class, and three class2 classes
+        // we want a duplicate varidx, a null varidx, and a device table?
+
+        fn make_val_record(
```

Comment:
Optional, this would be easier to read hoisted out like make_class_def. 

It's also hard to tell what the args are in the code. What if it took only the advance so this:

```rust
                make_val_record(5, None),
                make_val_record(6, DeviceOrVariationIndex::variation_index(4, 20)),
```

read more like this:

```rust
                val_with_xadvance(5),
                val_with_xadvance(6).with_x_advance_device(DeviceOrVariationIndex::variation_index(4, 20)),
```

make_class_def is similar, what's a class def 3,3,1?!

Reply:
This is defined inline since it's only used in this one test, whereas make_class_def is shared. :)